app/assets/javascripts/live_search.js

```diff
@@ -254,27 +294,30 @@
 
   LiveSearch.prototype.updateResultsCountMeta = function updateResultsCountMeta (totalCount) {
     // update search tracking meta data tag with new value
-    this.$resultsCountMetaTag.attr('content', totalCount)
+    if (this.$resultsCountMetaTag) {
+      this.$resultsCountMetaTag.setAttribute('content', totalCount)
+    }
   }
 
   LiveSearch.prototype.updateSortOptions = function updateSortOptions (results, action) {
-    if (action !== $.param(this.state)) { return }
+    if (action !== this.serializeState(this.state)) { return }
     this.updateElement(this.$sortBlock, results.sort_options_markup)
     this.bindSortElements()
   }
 
   LiveSearch.prototype.bindSortElements = function bindSortElements () {
-    this.$orderSelect = this.$form.find('.js-order-results')
-    this.$relevanceOrderOption = this.$orderSelect.find('option[value=' + this.$orderSelect.data('relevance-sort-option') + ']')
-    this.$relevanceOrderOptionIndex = this.$relevanceOrderOption.index()
+    this.$orderSelect = this.$form.querySelector('.js-order-results')
+    if (this.$orderSelect) {
+      this.$relevanceOrderOption = this.$orderSelect.querySelector('option[value=' + this.$orderSelect.getAttribute('data-relevance-sort-option') + ']')
+    }
   }
 
   LiveSearch.prototype.currentKeywords = function currentKeywords () {
     return this.getTextInputValue('keywords', this.state)
   }
 
   LiveSearch.prototype.updateOrder = function updateOrder () {
-    if (!this.$orderSelect.length) {
+    if (!this.$orderSelect || !this.$orderSelect.length) {
```

Comment:
I don't think this needs the 2nd half of the conditional. I can't see a scenario where this is a NodeList - perhaps I'm missing something?

Reply:
Good point, changed.